.github/ci_commit_pins/vision.txt

```diff
@@ -1 +1 @@
-01b9faa16cfeacbb70aa33bd18534de50891786b
+f36c5de4fdaa65b419babcbd4655f4c1a1d31ea6
```

Comment:
Is this done on purpose?

Reply:
Yes, this is the commit hash that my PR in torchvision https://github.com/pytorch/vision/pull/7625 was committed as. It's required to make CI pass